awscli/examples/cloudwatch/put-dashboard.rst

```diff
@@ -0,0 +1,15 @@
+**To create a dashboard**
+
+The following ``put-dashboard`` example creates a dashboard named ``Dashboard-A`` in the specified account. ::
+
+    aws cloudwatch put-dashboard \
+        --dashboard-name Dashboard-A \
+        --dashboard-body "{\"widgets\":[{\"type\":\"metric\",\"x\":0,\"y\":0,\"width\":6,\"height\":6,\"properties\":{\"view\":\"timeSeries\",\"stacked\":false,\"metrics\":[[\"Namespace\",\"CPUUtilization\",\"Environment\",\"Prod\",\"Type\",\"App\"]],\"region\":\"us-east-1\"}}]}"
```

Comment:
Command examples need to follow Linux quoting rules. The outside double quotes should be single quotes instead.